### ISSUE TITLE
Document TC39 communication tools

### ISSUE THREAD

**Issue by littledan** [tc39/how-we-work#5]
TC39 has a number of communication tools, for members and non-members, and it can be confusing to figure out how to access and use these. Although we have some explanations in CONTRIBUTING.md, I think things could be more clear. Some things which would be nice to have written somewhere include:
- Feedback about existing proposals should generally be given in bugs on GitHub repositories
- During meetings, there is the queue tool, and here's how to use it and what the various parts mean
- Delegates use the GitHub Reflector to communicate; make sure you're added to the Delegates or Invited Experts GitHub group (ask anyone in management to add you), and watch the Reflector repository.
- For TC39 delegates and invited experts, use the irc.freenode.net#tc39-delegates channel (include instructions for management to give people permission to join the channel, mention the use of NickServ)
- For anyone in the community to have interactive discussion, use the irc.freenode.net#tc39 channel (include explanation of how management should operate this, giving status to delegates)
- If es-discuss is referenced, include some sort of note of caution/discouragement

If he's available, I think @bterlson would be a great person to write this documentation, as he's been driving the engineering and operation of several of these tools over the years. This might make sense to divide into three different documents which are accessible to the relevant groups (community, delegates, and management). For now, we can draft this as a markdown file in this repo, while we work in parallel about the best way to surface it (#2).
